### ISSUE TITLE
safeAreaInsets broken

### ISSUE THREAD

**Issue by mariob** [schibsted/layout#97]
`safeAreaInsets` seems to be broken if using `LayoutLoading` instead of inheriting `LayoutViewController` pre 6.9.11 (tested in 6.9.10).

In 6.9.11, the `safeAreaInsets` seems to be always broken, no matter how you load the view. Doing a reload with Cmd-R seems to fix the safeAreaInsets.

Example:
The xml file looks like this:
```
<UIView backgroundColor="gray">
	<UIView backgroundColor="red" height="100" top="safeAreaInsets.top" />
</UIView>

```

A broken _view controller_:
```
class SafeAreaTestViewController: UIViewController, LayoutLoading {

    override func viewDidLoad() {
        super.viewDidLoad()

        loadLayout(named: "Example.xml")
    }
}
```

Result:
![broken](https://user-images.githubusercontent.com/75208/34650361-3efdd956-f3c0-11e7-857d-e546d6745d1b.png)

A working _view controller_:
```
class SafeAreaTestViewController: LayoutViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        loadLayout(named: "Example.xml")
    }
}

```

Result:
![working](https://user-images.githubusercontent.com/75208/34650365-4ec53a78-f3c0-11e7-819d-da4b0ad50295.png)

**Comment by nicklockwood** [comment]
Thanks, I’ll investigate. Probably a race condition, as safe area isn’t set up immediately when the app loads IIRC.

**Comment by nicklockwood** [comment]
@mariob can I confirm which version of iOS you're testing on? I can't reproduce this on iOS 11, but it seems to be happening on iOS 9.

**Comment by mariob** [comment]
iOS 11. First I noticed it on my device and then I tried it in the simulator. I'm pulling in Layout from Cocoapods.

Xcode Version 9.2 (9C40b).

Interesting observation from my _real_ project:  First the view is loaded incorrectly and after a short while the view is positioned correctly, just like if the safeAreaInsets was updated lika a state-change.

Let me check if I still have the _toy_ project on my drive. Maybe I can attach it so you can try it.

**Comment by nicklockwood** [comment]
@mariob can you check the version I just pushed to the develop branch? If that solves the issue for you then I'll tag it and push a new release.

**Comment by mariob** [comment]
@nicklockwood Looking good! Seems to work fine with your fix

**Comment by nicklockwood** [comment]
@mariob Thanks for confirming! Fixed in 0.6.12.
